adapter/config/parser.go

```diff
@@ -100,7 +101,13 @@ func ReadConfigs() (*Config, error) {
 			})
 			return
 		}
-
+		if invalidConfigError := adapterConfig.resolveJWTGeneratorConfig(); invalidConfigError != nil {
```

Comment:
This check only should happen if the backendJWT generation is enabled.

Reply:
ack